codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerBuilderGenerator.kt

```diff
@@ -151,6 +153,15 @@ class ServerBuilderGenerator(
         "MaybeConstrained" to RuntimeType.MaybeConstrained,
     )
 
+    fun composeDocAndCodeWritable(rustCrate : RustCrate, writer: RustWriter) {
+        val docWriter : () -> Unit = {
+            writer.docs("See #D.", structureSymbol)
+        }
+        rustCrate.withComposableInlineModule(builderSymbol.module(), codegenContext, docWriter) {
+            renderBuilder(this)
+        }
+    }
+
     fun render(writer: RustWriter) {
```

Comment:
I'd have guessed that this method would become `private` and tests should use `composeDocAndCodeWritable` too.

`composeDocAndCodeWritable` could simply be named `render`.

Reply:
I am with you on this. But right now, for the inline modules to be written out one has to call `RustCrate.renderComposableInlineModules`. Basically the inline writer needs to know that there are no more writers who would be writing to it, so that it can go ahead and render the registered `Writable`. At the moment, I couldn't figure out a better way of doing this than requiring someone to put an end to it by calling a function.

I will check what needs to be done in the tests for this to work.

Reply:
Oh yes. Let me fix this.